platform/migration/pom.xml

```diff
@@ -21,7 +21,7 @@
         <version>2.14.0-SNAPSHOT</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
-    <artifactId>migration</artifactId>
+    <artifactId>platform-migration-pom</artifactId>
```

Comment:
❓ thoughts on changing https://github.com/codice/ddf/blob/master/platform/migration/platform-migration/pom.xml to platform-migration-impl instead?

Reply:
I was avoiding changing artifactIds where it could cause a change of dependencies that are included elsewhere. Adding `-impl` would have larger / cascading effects where that jar is used.